OpenStack-Rabbit-Consumer/test/test_message_consumer.py

```diff
@@ -253,25 +246,94 @@ def test_consume_create_machine_hostnames_good_path(
 
 
 @patch("rabbit_consumer.message_consumer.delete_machine")
-def test_consume_delete_machine_good_path(
-    delete_machine, rabbit_message, openstack_address_list
-):
+def test_consume_delete_machine_good_path(delete_machine, rabbit_message):
     """
     Test that the function calls the correct functions in the correct order to delete a machine
     """
     rabbit_message.payload.metadata.machine_name = "AQ-HOST1"
 
-    with (
-        patch("rabbit_consumer.message_consumer.VmData") as data_patch,
-        patch("rabbit_consumer.message_consumer.openstack_api") as openstack,
-    ):
-        openstack.get_server_networks.return_value = openstack_address_list
-
+    with (patch("rabbit_consumer.message_consumer.VmData") as data_patch,):
```

Comment:
Why are we doing a tuple here?

Reply:
It was from when we patched more than one thing, I'll fix this now :+1: